string filename
---
# Result code defintions
uint8 RESULT_SUCCESS=0
uint8 RESULT_FAILED_TO_WRITE_FILE=255

uint8 result